### PR TITLE
Jia ping1019 taiwan

### PR DESCRIPTION
**Pull request by JiaPing1019**
Add my country
@githubteacher
closes #2450

### CODE REVIEW

File: test [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (jr-cologne) [inline test]: Delete this file

File: _pins/JiaPing.json. [hunk 2]
```diff
@@ -0,0 +1,3 @@
+githubHandle: JIAPING
```
Comment (jr-cologne) [inline _pins/JiaPing.json.]: The syntax of the file is wrong.

It should look like this:

````
---
githubHandle: JiaPing1019
latitude: 25.016983
longitude: 121.462787
---
````

Comment (JiaPing1019) [inline _pins/JiaPing.json.]: Thank you

Comment (jr-cologne) [inline _pins/JiaPing.json.]: You're welcome. Let me know, if you need any further help.

**Comment by awbloss** [comment]
Hi JiaPing1019,

Is your pin on the map?

Andrew

**Comment by brianamarie** [comment]
Hi @JiaPing1019! I went ahead and fixed the syntax of your file. I'm going to merge this now since it doesn't look like you've been back in a few days and I'm one of the maintainers of this repository. Awesome work!

**Pull request merged**